### PR TITLE
Add Chinese translation of IBC architectural overview and ICS 02.

### PR DESCRIPTION
**Pull request by arthaszeng**
Add Chinese translation of IBC architectural overview and ICS 02.

### CODE REVIEW

**Comment by cwgoes** [comment]
Do you want this to be reviewed as-is or do you want to add more first?

**Comment by arthaszeng** [comment]
> Do you want this to be reviewed as-is or do you want to add more first?

Hi cwgoes, I think we can let it be reviewed for now as a start, thus other Chinese developers who are interested with Cosmos like me could contribute to the translation work. Also I will keep going to translate other parts continuously. If you have any concern or suggestion pls feel free to leave me a msg.

File: translation/zh-cn/ibc-overview.md [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+
+### **Actor / 参与者**
+
+角色或用户是在跨链通信中进行交互的实体。一个参与者可以是一个终端用户，在区块链上运行的模块或智能合约，或者是能够签署交易的链下中继器进程。
+
+### **Machine / Chain / Ledger / 机器 / 链 / 账本**
+
+机器，链，块链或账本（可互换使用）是实现IBC规范的一部分或全部的状态机（并不需要严格意义上的区块链，可以是分布式账本或者链）。
+
+### **Relayer process / 中继链进程**
```
Comment (en) [inline translation/zh-cn/ibc-overview.md]: 中继链 should refer to *relay chain*, *relayer process* is best translated into Chinese as 中继进程.

Comment (arthaszeng) [reply translation/zh-cn/ibc-overview.md]: Thanks for advice, fixed.

Comment (arthaszeng) [reply translation/zh-cn/ibc-overview.md]: @en Hi Yuanchao, is there any other comments or advices? Could you please tell me any assistance I could provide before merging this? Thank you.

Comment (en) [reply translation/zh-cn/ibc-overview.md]: hi @arthaszeng, during the review, I also found some nits should be fixed. However, according to @Satoshi-Kusumoto 's suggestion in #253 , we'd better use GitLocalize to translate (as Korean version). I think this PR is a good start, we should merge it first, then I will open a new PR to migrate your translation to GitLocalize.

**Comment by arthaszeng** [comment]
Hi @en , that's a good suggestion, I agree to merge this as a start. 
I also took a look around GitLocalize tool which does look good for me, I'm happy to use it  in further translation stuff. 😃

**Comment by cwgoes** [comment]
Thanks @en for the review.

**Pull request merged**